### PR TITLE
🔨 Removing cocos renderer files

### PR DESCRIPTION
**Pull request by HarsimranVirk**
Solves #2863

### CODE REVIEW

File: newIDE/app/src/Export/ExportDialog.js [hunk 1]
```diff
@@ -211,23 +243,11 @@ export default class ExportDialog extends React.Component<Props, State> {
...
-                            primary={false}
-                            onClick={() => this._showExperimental(false)}
-                            label={<Trans>Hide experimental exports</Trans>}
-                          />
-                        )}
+                        <ExperimentalExportButton
+                          showExperimental={showExperimental}
+                          onClick={value => this._showExperimental(value)}
+                          disabled
+                        />
```
Comment (HarsimranVirk) [inline newIDE/app/src/Export/ExportDialog.js:250]: Extracted this out, as at the moment, cocos2d was the only experimental export. If in future we decide to add more experimental exports, this should come in handy.

**Review by 4ian — APPROVED** [review]
I think this looks good! 

Before we go ahead, can you check:
- You can export a game (try the Android one click packaging, web export and the Windows export for example).
- No bug/crash on a few different examples

Just want to be sure we've not removed something that was necessary somewhere, but I think it's fine! :)

**Comment by Bouh** [comment]
- Android one click, [web export](https://games.gdevelop-app.com/game-6fd11cdb-a3ad-4aac-a58c-bea054be49d9/index.html) and windows export are ok.
- No bug on examples.

 .map files are missing in web export but seem not related to this PR.


It's a go!

**Comment by 4ian** [comment]
> .map files are missing in web export but seem not related to this PR.

Indeed, unrelated to the PR (and harmless, though not having these warning would be better).

Merging then! 🎉

**Pull request merged**